tests/unit/core/metrics/test_model_metrics.py

```diff
@@ -60,17 +60,36 @@ def tests_no_metrics_to_protobuf_regression():
     model_metrics = ModelMetrics.from_protobuf(message)
     assert model_metrics.model_type == ModelType.REGRESSION
 
+def regression_metrics_eq(self, other):
```

Comment:
This might be something we get using dataclass as a base. We should think of adding this later to all objects, will make testing smoother.

Reply:
excellent, let me try marking regression metric as a @dataclass.